vhdbuilder/scripts/linux/mariner/tool_installs_mariner.sh

```diff
@@ -138,3 +138,28 @@ activateNfConntrack() {
     # explicitly activate nf_conntrack module so associated sysctls can be properly set 
     echo nf_conntrack >> /etc/modules-load.d/contrack.conf
 }
+
+installFIPS() {
+
+    echo "Installing FIPS..."
+
+    # Install necessary rpm pacakages
+    dnf_install 120 5 25 grubby || exit $ERR_APT_INSTALL_TIMEOUT
+    dnf_install 120 5 25 dracut-fips || exit $ERR_APT_INSTALL_TIMEOUT
+    dnf_install 120 5 25 python3-dnf-plugin-versionlock || exit $ERR_APT_INSTALL_TIMEOUT
```

Comment:
If we're not locking package versions then we don't need the versionlock plugin.
```suggestion
```